pulse/interface/main_window.py

```diff
@@ -580,26 +580,45 @@ def action_clip_plane_callback(self):
         self.clip_plane.slider_released.connect(self.apply_clip_plane)
         self.clip_plane.closed.connect(self.close_clip_plane)
 
+    def action_zoom_callback(self):
+        if self.get_current_workspace() == 0:
+            self.opv_widget.opvGeometryRenderer.resetCamera()
+            self.update()
+        
+        elif self.get_current_workspace() == 1:
+            self.opv_widget.opvRenderer.resetCamera()
+            self.opv_widget.opvRenderer.update()
+    
+        elif self.get_current_workspace() == 3:
+            self.opv_widget.opvAnalysisRenderer.resetCamera()
+            self.opv_widget.opvAnalysisRenderer.update()
+            self.opv_widget.opvRenderer.resetCamera()
+            self.opv_widget.opvRenderer.update()
+
+        else:
+            self.opv_widget.opvRenderer.resetCamera()
+            self.opv_widget.opvRenderer.update()
+        
     def set_clip_plane_configs(self):
-        if self.combo_box_workspaces.currentText() == "Results Workspace":
+        if self.get_current_workspace() == 3:
```

Comment:
Do not use the number 3 directly in this comparison, use ```Workspace.RESULTS``` instead.
It makes a bit more clear wich workspace are you testing.